src/main/java/org/folio/circulation/services/AllowedServicePointsService.java

```diff
@@ -145,20 +145,25 @@ private CompletableFuture<Result<User>> fetchUser(AllowedServicePointsRequest re
   private CompletableFuture<Result<Map<RequestType, Set<AllowedServicePoint>>>>
   getAllowedServicePoints(AllowedServicePointsRequest request, User user, Collection<Item> items) {
 
+    String indexName = request.isEcsRequestRouting()
+      ? "ecsRequestRouting"
+      : "pickupLocation";
```

Comment:
So we are now searching either by `ecsRequestRouting` or `pickupLocation`. Do we not care about the value of `pickupLocation` when fetching ECS routing service points?